### ISSUE TITLE
Change default high contrast version colours

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1100]
The high contrast default Orange and yellow colours used, despite contrasting against the black background, do not contrast against each other, meaning where visited and un-visited links are in close proximity (e.g. goals page), they are difficult to tell apart for colour blind users.

The following is therefore proposed:

Current: Orange (#F47738), Yellow (#FFFF00)

Proposed: Orange (#AD6800), Yellow (#FFDD00)
![Screenshot (145)](https://user-images.githubusercontent.com/60919820/108836410-b4abd080-75c8-11eb-872c-74a94f93ce7f.png)

**The new colours would also need to be reflected on the ‘A’ icons.**
